.github/workflows/ci.yml

```diff
@@ -43,6 +43,19 @@ jobs:
       - uses: dtolnay/rust-toolchain@nightly
       - run: cargo test
 
+  windows:
+    name: Windows
+    needs: pre_ci
+    if: needs.pre_ci.outputs.continue
+    runs-on: windows-latest
+    timeout-minutes: 45
+    steps:
+      - uses: actions/checkout@v3
+      - uses: dtolnay/rust-toolchain@nightly
+      - run: cargo build
+      - run: ls target/debug
```

Comment:
my bad for these extra steps, thanks for removing in a87f7f20e2593865ce116824bf92922fe5ada6e1